packages/sharing-smart-contract/test/invariant/Invariant.t.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.24;
+
+import {StdInvariant} from "forge-std/StdInvariant.sol";
+import {Test} from "forge-std/Test.sol";
+import {HandlerSale} from "./handlers/HandlerSale.sol";
+import {HandlerSubscription} from "./handlers/HandlerSubscription.sol";
+import {HandlerRenting} from "./handlers/HandlerRenting.sol";
+
+contract Invariant is StdInvariant, Test {
+    function setUp() public {
+        vm.createSelectFork("https://bellecour.iex.ec");
+        HandlerSubscription hSubscription = new HandlerSubscription();
+        targetContract(address(hSubscription));
+
+        vm.createSelectFork("https://bellecour.iex.ec");
```

Comment:
a single `vm.createSelectFork` should be sufficient, no need to call it multiple times